src/profile/script.js

```diff
@@ -67,9 +74,10 @@ function createCard({ oldData, newData, userId, username, profileDiffId }) {
   //looping through the old data to display in list
   for (const listItem in oldData) {
     const li = document.createElement('li');
-    li.innerText = `${formatPropertyField(listItem)}: ${
-      oldData[listItem] || (listItem === 'yoe' ? NaN : '')
-    }`;
+    li.innerText = `${formatPropertyField(listItem)}: ${getDataItem(
```

Comment:
This can be a util function. Instead of writing it 2 times. Make a util function and re-use it.